src/components/MarketDetail/index.js

```diff
@@ -104,7 +104,25 @@ class MarketDetail extends Component {
       marketFetchError: undefined,
     }
   }
+
   componentWillMount() {
+    if (!this.props.params.view) {
+      this.props.changeUrl(`/markets/${this.props.params.id}/${DEFAULT_VIEW}`)
+    }
+
+    this.fetchEssentialData()
+    this.props.requestGasCost(GAS_COST.BUY_SHARES)
+    this.props.requestGasCost(GAS_COST.SELL_SHARES)
+
+    this._fetchDataTimer = setInterval(this.fetchEssentialData, 15000)
```

Comment:
15 seconds? @giacomolicari is that ok?

Reply:
This can be parameterized inside config.json:
- fetchMarketTimeInterval: 150000